configs/pontoon.toml

```diff
@@ -377,3 +377,16 @@ locales = [
 [[paths]]
     reference = "en/products/vpn/features.ftl"
     l10n = "{locale}/products/vpn/features.ftl"
+[[paths]]
```

Comment:
Move this to where the whatsnew section is in alphabetical order so it's easy to locate it when it's time to remove the entry that's no longer used.